manifest.json

```diff
@@ -346,48 +336,29 @@
         "rootDirPrefix": "guppy-",
         "targetDir": "guppy-b5055b"
       },
-      "hammerJs": {
-        "version": "2.0.8",
-        "downloadFormat": "zip",
-        "url": "https://github.com/hammerjs/hammer.js/archive/v2.0.8.zip",
-        "rootDirPrefix": "hammer.js-",
-        "targetDirPrefix": "hammer-js-",
-        "bundle": {
-          "js": ["hammer.js"]
-        }
-      },
       "headroomJs": {
         "version": "0.9.4",
         "downloadFormat": "files",
         "url": "https://cdnjs.cloudflare.com/ajax/libs/headroom/0.9.4",
         "rootDirPrefix": "headroom-js-",
         "targetDirPrefix": "headroom-js-",
-        "files": ["angular.headroom.min.js", "headroom.min.js"],
-        "bundle": {
-          "js": []
-        }
+        "files": ["angular.headroom.min.js", "headroom.min.js"]
```

Comment:
For this, shouldn't `headroom.min.js` come before `angular.headroom.min.js` or does the order not matter?

Reply:
These are just the files that are placed in some folder so the order doesn't matter.